ntp-proto/src/nts_record.rs

```diff
@@ -217,33 +254,23 @@ impl NtsRecord {
 
     #[allow(unused_variables)]
     pub fn client_key_exchange_records(
-        ntp_version: Option<NtpVersion>,
+        ntp_version: ProtocolVersion,
         denied_servers: impl IntoIterator<Item = String>,
     ) -> Box<[NtsRecord]> {
         let mut base = vec![
-            #[cfg(feature = "ntpv5")]
             NtsRecord::DraftId {
                 data: crate::packet::v5::DRAFT_VERSION.as_bytes().into(),
             },
```

Comment:
This record should only be included if we ask for ntpv5

Reply:
Fixed in eff750520a38376b666d24858b8630cdd66fb8be